### PR TITLE
Fixed usability issues on submission intake form and other proponent pages

### PR DESCRIPTION
**Pull request by qhanson55**
# Description

Hid location coordinate helper texts when proponent selects the street address radio button. 
Removed the field checkProvincialPermits and all references from the frontend and app.
Added back button to the My drafts and submissions page for proponents.
Removed spatial file upload accordion and "added" it to the upload file section in the housing tab.
Changed back button on intake form to go back to drafts and submissions if the proponent clicked into a draft and left it to take you back to housing if it's a new form.
Stopped the intake form from autosaving when changing tabs on a blank intake form

This will help with the usability of the application for proponents.
[PADS-280](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-280)

## Types of changes

Bug fix (non-breaking change which fixes an issue)

Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Subin1Doo** [comment]
on the "my drafts and previous entries page" the "Back to Housing" button should go above the page title, below the top menu bar. Also, on the submitted/ saved draft enquiries, the back button should be changed to "back to my drafts and previous entries." Thank you!

File: frontend/src/components/housing/submission/SubmissionIntakeSchema.ts [hunk 1]
```diff
@@ -153,14 +153,6 @@ export const submissionIntakeSchema = object({
     ltsaPIDLookup: string().max(255).nullable().label('Parcel ID'),
     geomarkUrl: string().max(255).label('Geomark web service url')
   }),
-  [IntakeFormCategory.PERMITS]: object({
-    hasAppliedProvincialPermits: string().oneOf(YES_NO_UNSURE_LIST).required().label('Applied permits'),
```
Comment (kyle1morel) [inline frontend/src/components/housing/submission/SubmissionIntakeSchema.ts:157]: I think we still want this bit in, doesn't look like it was removed from the form.

File: frontend/src/components/housing/submission/SubmissionIntakeForm.vue [hunk 2]
```diff
@@ -731,8 +746,17 @@ onBeforeMount(async () => {
...
+                  <label class="col-12">
+                    Upload documents about your housing project (pdfs, maps,
+                    <a
+                      href="https://portal.nrs.gov.bc.ca/documents/10184/0/SpatialFileFormats.pdf/39b29b91-d2a7-b8d1-af1b-7216f8db38b4"
+                      target="_blank"
+                      class="text-blue-500 underline"
+                    >
+                      {{ 'shape files' }}
+                    </a>
+                    {{ 'etc)' }}
```
Comment (wilwong89) [inline frontend/src/components/housing/submission/SubmissionIntakeForm.vue:758]: Is there a need to string interpolate a string here?

File: frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue [hunk 3]
```diff
@@ -81,8 +81,23 @@ const formSchema = object({
...
+      routeName: RouteName.HOUSING_SUBMISSIONS
+    };
+  } else {
+    return {
+      text: 'Back to Housing',
+      routeName: RouteName.HOUSING
+    };
+  }
+});
+
```
Comment (wilwong89) [inline frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue:100]: This doesn't appear to be functional? I am unable to trigger the first conditional to get "Back to my drafts..." even when navigating to the form from a draft enquiry.

File: frontend/src/components/housing/submission/SubmissionIntakeForm.vue [hunk 4]
```diff
@@ -127,6 +129,20 @@ const { formUpdated, stopAutoSave } = useAutoSave(() => {
...
+      routeName: RouteName.HOUSING_SUBMISSIONS
+    };
+  } else {
+    return {
+      text: 'Back to Housing',
+      routeName: RouteName.HOUSING
+    };
+  }
+});
+
```
Comment (wilwong89) [inline frontend/src/components/housing/submission/SubmissionIntakeForm.vue:145]: This doesn't appear to be functional? I am unable to trigger the first conditional to get "Back to my drafts..." even when navigating to the form from a draft submission.

**Review by kyle1morel — APPROVED** [review]

**Pull request merged**